src/lib/netlist/plib/ptypes.h

```diff
@@ -15,35 +15,34 @@
 #include <type_traits>
 
 #if (PUSE_FLOAT128)
-#if defined(__has_include)
-#if __has_include(<quadmath.h>)
-#include <quadmath.h>
-#endif
-#endif
+	#if defined(__has_include)
+		#if __has_include(<quadmath.h>)
+			#include <quadmath.h>
+		#endif
+	#endif
 #endif
 
-#define PCOPYASSIGNMOVE(name, def) \
-	PCOPYASSIGN(name, def) \
+#define PCOPYASSIGNMOVE(name, def)                                             \
+	PCOPYASSIGN(name, def)                                                     \
 	PMOVEASSIGN(name, def)
 
-#define PCOPYASSIGN(name, def)  \
-	name(const name &) = def; \
+#define PCOPYASSIGN(name, def)                                                 \
+	name(const name &) = def;                                                  \
 	name &operator=(const name &) = def;
 
-#define PMOVEASSIGN(name, def)  \
-	name(name &&) noexcept = def; \
+#define PMOVEASSIGN(name, def)                                                 \
+	name(name &&) noexcept = def;                                              \
```

Comment:
What’s the benefit of pushing the preprocessor continuation backslashes so far to the right?  There’s nothing in this group of macros that uses that much space.